### DISCUSSION TITLE
ALM TestSet{i} dynamic

### DISCUSSION THREAD

**Discussion by vivekabanda** [community/community#31407] [Other Feature Feedback, Questions, & Ideas]
MicroFocus / ADM-FT-ToolsLauncher - under Sample 4: Run ALM test sets, can we make TestSet{i} dynamic instead of giving test set name manually like test set 1= <folder name>

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/MicroFocus/ADM-FT-ToolsLauncher? In that case you should open an issue *in that repository* so the developers see it. :slightly_smiling_face:

**Reply by vivekabanda** [reply]
Its not an issue, I'm looking if there a code to make dynamic

**Reply by airtower-luna** [reply]
Either way you're much more likely to get a helpful response if you ask people who actually know the software. It's possible someone here does, but it'd be luck. :slightly_smiling_face:
